plugin/src/main/java/me/champeau/mrjar/MultiReleaseExtension.java

```diff
@@ -117,6 +117,7 @@ private void addLanguageVersion(int version) {
         testImplementation.extendsFrom(configurations.getByName(sharedTestSourceSet.getImplementationConfigurationName()));
         Configuration testCompileOnly = configurations.getByName(testSourceSet.getCompileOnlyConfigurationName());
         testCompileOnly.extendsFrom(configurations.getByName(sharedTestSourceSet.getCompileOnlyConfigurationName()));
+        testCompileOnly.getDependencies().add(dependencies.create(langSourceSet.getOutput().getClassesDirs()));
```

Comment:
mmm, I think this should be `testImplementation`, not `testCompileOnly`, right?

Reply:
I think this dependency isn't needed at runtime, since the full JAR file is used instead.